### ISSUE TITLE
Discovery:  Pull a mobile component from their instance Storybook into DST Storybook

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#2392]
## Description
We need to try pulling in a mobile component from the mobile teams instance of Storybook into the DS instance of Storybook. 
- Create a section for Mobile and put all mobile button component in that section. 
- Pull in the Button component. 

Coordinate with the mobile team on how to pull one of their components into DS Storybook if needed. 

## Tasks
- [ ] Work with mobile team dev to access their instance of Storybook
- [ ] Add a mobile component section in DS storybook 
- [ ] Add or pull in the mobile button component into the mobile section 
- [ ] Review with PO and Designers

## Acceptance Criteria
- [ ] A mobile component section has been added to the VA DS Storybook instance
- [ ] The mobile button component is visible to the mobile section

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/2392?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @nickjg231 @powellkerry @rmessina1010 @rsmithadhoc

**Comment by danbrady** [comment]
FYI: [Mobile Storybook](https://department-of-veterans-affairs.github.io/va-mobile-library/?path=/docs/button--docs)

**Comment by it-harrison** [comment]
I was able to incorporate the Mobile team's storybook via a simple update to our storybook configuration to use [composition](https://github.com/department-of-veterans-affairs/va.gov-team/issues/92340). This pulls in the entire Mobile team storybook, which is presumably what we want, although this ticket specified bringing in a single component.

I created a [draft PR](https://github.com/department-of-veterans-affairs/component-library/pull/1314) whose chromatic instance demonstrates the result:

<img width="1460" alt="Screenshot 2024-09-06 at 10 30 18 AM" src="https://github.com/user-attachments/assets/fe1444ad-538a-4b38-9107-5ccc3e6caf06">

In summary, it seems like it will be easy to incorporate the Mobile Team's storybook instance, _**so long as they continue to deploy their storybook instance to a publicly accessible url**_.

**Comment by jamigibbs** [comment]
@it-harrison Very cool find!! I'm also assuming we will be ok with displaying all of the components but just in case, does this composition setting allow us to filter out individual components so that they don't display? Or maybe we would just need to do that with some custom scripting?

**Comment by it-harrison** [comment]
If we need to filter stories from `va-mobile`'s storybook this is not something that storybook supports out of the box; it looks to be possible to do by writing a custom addon. However, @humancompanion-usds indicated during demo that this is not required because we want to bring in all the stories.
